docs/en/advanced_tutorials/registry.md

```diff
@@ -75,7 +77,7 @@ print(ACTIVATION.module_dict)
 ```
 
 ```{note}
-The registry mechanism will only be triggered when the corresponded module file is imported, so we need to import the file somewhere or dynamically import the module using the ``custom_imports`` field to trigger the mechanism. Please refer to [Importing custom Python modules](config.md#import-the-custom-module) for more details.
+The registry mechanism will only be triggered when the corresponded module file is imported, so we need to make the registry automatically import the module through the predefined `locations`, or import the file somewhere manually, or dynamically import the module using the ``custom_imports`` field to trigger the mechanism. Please refer to [Importing custom Python modules](config.md#import-the-custom-module) for more details.
```

Comment:
It would be better to use bullet points, an example i like below:

The key to trigger the registry mechanism is to make the module imported. 
There are three ways to register a module into the registry
1. implement the module in the 'locations' and make sure it is imported. The registry will automatically import modules ib the predefined locations, xxxxx. This is to ease the usage of algorithm libraries so that users can directly use 'REGISTRY.build(cfg)'
2 import the file manually. This is common when developers implement a new module in/out side the algorithm library. xxxx
3 usr custom_imports in config.. xxxx